lib/classes/Database.js

```diff
@@ -15,31 +15,61 @@ class Database {
 
 	static async initialize() {
 		log("log", "Initializing database...");
+
 		const configDirectory = await fs.readdir(process.cwd() + "/config");
-		if (!configDirectory.includes("auth.json")) {
-			log("error", "Initialization failed! Authorization file (/config/auth.json/) not found!");
-			process.exit();
+		if (!configDirectory.includes("db.json")) {
+			log("warn", "Can not load database configuration file. Using sqlite in-memory database as fallback option.");
+			this.sequelizeInstance = this.#dbInstance.inmemory();
+		} else {
+			let dbConfig = require("../../config/db.json");
+			let dbtype = dbConfig.dbtype;
+
+			this.sequelizeInstance = this.#dbInstance[dbtype](dbConfig);
```

Comment:
File checking is ok, but here one problem left. If `dbType` will be not equal to `inmemory` or `mariadb`, then this line will throw `TypeError`. Need to add optional chaining:

```js
this.sequelizeInstance = this.#dbInstance[dbType]?.(dbConfig) ?? this.#dbInstance[defaultDatabaseTypeHere](dbConfig);
```

If there no function with `dbType` name then it will return `undefined`. If undefined returned then call `defaultDatabaseTypeHere` function.

Reply:
@koloml can you please resolve with this issue by yourself and then accept PR, so you can continue development. At the moment I can’t temporarily take part in the development for several days.

Reply:
Yup. See ya later.